hole/poker.go

```diff
@@ -5,7 +5,27 @@ import (
 	"slices"
 )
 
-func cardRune(number, suit int) rune {
+type CardValue int
```

Comment:
So this makes it public in Go, lowercase would be private to just the "hole" namespace. So make it "cardValue".